engine/model_transactions.go

```diff
@@ -240,8 +240,6 @@ func (m *Transaction) setChainInfo(txInfo *chainstate.TransactionInfo) {
 // Converts from bc.BUMP to our BUMP struct in Transaction model
 func (m *Transaction) setBUMP(txInfo *chainstate.TransactionInfo) {
 	switch {
-	case txInfo.MerkleProof != nil:
```

Comment:
I'm wondering if this switch-case is still necessary. Maybe just `if-else` would be more straightforward.

Reply:
changed to `if-else`